src/ansys/fluent/core/launcher/launch_options.py

```diff
@@ -294,6 +302,22 @@ def _get_graphics_driver(
         if is_windows()
         else FluentLinuxGraphicsDriver(graphics_driver)
     )
+    if graphics_driver.value != "null" and ui_mode_ not in {
+        UIMode.GUI,
+        UIMode.HIDDEN_GUI,
+    }:
+        warnings.warn(
+            """\nIf the UI mode is ``UIMode.GUI`` or ``UIMode.HIDDEN_GUI``,
```

Comment:
I don't think there is any restriction for setting graphics driver in `GUI` and `HIDDEN_GUI` mode.

We want warn user if they specify graphics driver in other modes.